examples/avmnist_multimodal/README.md

```diff
@@ -0,0 +1,24 @@
+# Multimodal Digit Classification: Multimodal Deep Learning model (MMDL) on AVMNIST
+
```

Comment:
Can you give a little bit more background info about the data please? Not everyone can learn from this brief ReadMe.

Reply:
Background info about the data as been added.